src/server/service/kv.rs

```diff
@@ -1824,6 +1824,13 @@ fn extract_region_error<T>(res: &storage::Result<T>) -> Option<RegionError> {
             err.set_server_is_busy(server_is_busy_err);
             Some(err)
         }
+        Err(storage::Error::Closed) => {
```

Comment:
any test to cover this?

Reply:
`Error::Closed` is enough?

Reply:
We have a long time running schrodinger test to cover this scenario.